Extension/package.json

```diff
@@ -3280,9 +3280,20 @@
                 "default": "exec-run"
               },
               "visualizerFile": {
-                "type": "string",
```

Comment:
You will also want to update https://github.com/microsoft/vscode-cpptools/blob/main/Extension/tools/OptionsSchema.json

Then run `yarn generateOptionsSchema` since this section of the package.json is auto-generated by [GenerateOptionsSchema.ts](https://github.com/microsoft/vscode-cpptools/blob/main/Extension/tools/GenerateOptionsSchema.ts)